### PR TITLE
pass namespace via action_env, and use multirun

### LINKED ISSUE #7: [Refactor] Use multirun instead of bazel query to push/load all images
Credit: thanks @rdzhou for bringing this up.

> Maybe [use multirun](https://github.com/bazel-contrib/rules_oci/blob/main/docs/push.md#examples) to create a "push_all" target instead of `bazel query 'kind("oci_push", "//...")' | xargs -n1 bazel run`, same for oci_load

I would create multirun rules `push_all_images` and `load_all_images`.

### PR DESCRIPTION
**Pull request by dayeol**
instead of a shell script

This resolves #7

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -42,32 +34,43 @@ genrule(
...
+        outs = ["{}_repo.txt".format(k)],
+        cmd = "echo '{}' | envsubst > $@".format(v)
+    )
+    for (k, v) in REPOS.items()
+]
+[
+    oci_push(
+        name = "push_{}_image".format(k),
+        image = "//app/{}:image".format(k),
+        remote_tags = ["lIatest"],
```
Comment (privacywill) [inline BUILD:56]: lIatest  Is it a typo?

Comment (dayeol) [reply BUILD]: yeah. fixed!

File: BUILD [hunk 2]
```diff
@@ -42,32 +33,43 @@ genrule(
...
+    oci_push(
+        name = "push_{}_image".format(k),
+        image = "//app/{}:image".format(k),
+        remote_tags = ["latest"],
+        repository_file = ":{}_repo".format(k)
+    )
+    for (k, v) in REPOS.items()
+]
+[
+    command(
```
Comment (dayeol) [inline BUILD:61]: actually, I may not need this. I'll remove it.

**Review by rdzhou — APPROVED** [review]
LGTM

**Pull request merged**